src/util/ALE.sol

```diff
@@ -210,27 +200,52 @@ contract ALE is Ownable, ReentrancyGuard, CurveDBRHelper {
 
         // If there's an helper contract, the collateral has to be transformed
         if (address(markets[_sellTokenAddress].helper) != address(0)) {
+            uint256 estimateAmount = markets[_sellTokenAddress]
+                .helper
+                .collateralToAsset(_collateralAmount);
+
             // Collateral amount is now transformed into sellToken
             _collateralAmount = markets[_sellTokenAddress]
                 .helper
                 .transformFromCollateral(_collateralAmount, _helperData);
+
+            if (
+                _collateralAmount + 1 < estimateAmount &&
```

Comment:
What's the + 1 here?

Reply:
The estimation was not so precise, update the calculation in the helper contract with same logic yearn uses, now `_collateralAmount` matches exactly `estimateAmount`, fixed in [21f4846](https://github.com/InverseFinance/FiRM/commit/21f4846305105726621e881e3f1a85110a3d3dcd)